sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -1707,7 +1708,7 @@ def stage_block_from_url(
             source_content_md5=source_content_md5,
             **kwargs)
         try:
-            self._client.block_blob.stage_block_from_url(**options)
+            return self._client.block_blob.stage_block_from_url(**options)
```

Comment:
Would you like to also help teak the doc a bit here ^_^
1. for line 1674 would you like to make it from `# type: (...) -> None` to `# type: (...) -> Dict[str, Any]`
2. for line 1701 would you like to make it
```
:returns: Blob property dict
:rtype: dict[str, Any]
```

We are really appreciated for your help again!

Reply:
Done